internal/backend/db/dbgorm/gorm_test.go

```diff
@@ -45,7 +47,7 @@ func setupDB(dbName string) *gorm.DB {
 		},
 	)
 	if dbName == "" {
-		dbName = ":memory:"
+		dbName = "file::memory:?cache=shared"
```

Comment:
why do you need to share stuff between tests ?

Reply:
fixed. It seems the problem was with transactions. They were opening new db connection (closing old one), which was causing to all data to disappear in db. Once we got rid of transactions, the problem was gone